infra/base-images/base-builder/cargo

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# we need this until https://github.com/rust-lang/cargo/issues/5450 or similar is merged
+# because cargo uses relative paths for the current crate
+# and absolute paths for its dependencies
+
```

Comment:
Hack is to use this cargo which come first in `PATH` 
So we can change `RUSTFLAGS` for this execution
then execute `/rust/bin/cargo`